types/inline-style-prefixer/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for inline-style-prefixer 3.0
+// Project: https://github.com/rofrischmann/inline-style-prefixer
+// Definitions by: Andrej Hazucha <https://github.com/ahz>
+//                 dpetrezselyova <https://github.com/dpetrezselyova>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default InlineStylePrefixer;
```

Comment:
I've been considering this a lot. Because if I look correctly distributed version of the npm module has both commonjs exports and also ES2015 default export.

```js
exports.default = Prefixer;
module.exports = exports['default'];
```

Am I correct? What is the preference if modules supports both `export= ` and `export default`? In the sources there is [default export](https://github.com/rofrischmann/inline-style-prefixer/blob/master/modules/dynamic/index.js#L37).